src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.Resources.AllLangsNoEnu.swr

```diff
@@ -0,0 +1,8 @@
+folder "InstallDir:\MSBuild\15.0\Bin\Roslyn\$(Culture)"
+  file source=$(OutputPath)\Dlls\Microsoft.CodeAnalysis\$(Culture)\Microsoft.CodeAnalysis.resources.dll
+  file source=$(OutputPath)\Dlls\Microsoft.CodeAnalysis.CSharp\$(Culture)\Microsoft.CodeAnalysis.CSharp.resources.dll
+  file source=$(OutputPath)\Dlls\Microsoft.CodeAnalysis.VisualBasic\$(Culture)\Microsoft.CodeAnalysis.VisualBasic.resources.dll
+  file source=$(OutputPath)\Dlls\Microsoft.CodeAnalysis.Scripting\$(Culture)\Microsoft.CodeAnalysis.Scripting.resources.dll
+  file source=$(OutputPath)\Dlls\Microsoft.CodeAnalysis.CSharp.Scripting\$(Culture)\Microsoft.CodeAnalysis.CSharp.Scripting.resources.dll
+  file source=$(OutputPath)\Dlls\Microsoft.Build.Tasks.CodeAnalysis\net46\$(Culture)\Microsoft.Build.Tasks.CodeAnalysis.resources.dll
```

Comment:
What validation do we get when these entries are wrong? For instance the net46 folder will change very soon due to Dev16 changes. Will the build break when there is no longer output to the net46 folder (hoping the answer is yes)?